Allow grayscale option for plot.FFTrees()

ndphillips opened the pull request:
A user Guido Williams <guido.l.williams@gmail.com> emailed me asking for a grayscale option for plot.FFTrees().

>Dear Nathaniel,
>
>I have a question regarding the plot functions in the FFT package. Is there a simple way to print all plots/output in black and white/greyscale? For instance: plot(heart.fft, color = FALSE) or is there another way?
This would be very convenient for publication purposes.


This PR includes the new `grayscale` argument to `plot.FFTrees()` that enables this

See example below:

```
heart_fft <- FFTrees(formula = diagnosis ~.,
                     data = heart.train,
                     data.test = heart.test, 
                     decision.labels = c("Healthy", "Disease"))


plot(heart_fft, grayscale = TRUE)
```

<img width="523" alt="image" src="https://github.com/ndphillips/FFTrees/assets/8480980/78c3d1d3-3200-46c8-980d-fcab088f21b3">


Setting `grayscale = FALSE` (the default), gives us the color back

```
plot(heart_fft, grayscale = FALSE)
```

<img width="522" alt="image" src="https://github.com/ndphillips/FFTrees/assets/8480980/e51dd991-82f0-41e6-bbf8-35a1fc10edea">

ndphillips commented:
Guido emailed me to confirm it works as desired:

> It works like a charm! Thanks!

hneth commented:
This seems to be a nice enhancement and useful option — **thanks** for suggesting and implementing it!

An alternative approach to colors in FFT plots could collect all used ones in a vector and pass that as an argument. This would provide users with complete control over the colors, but the combination of a good default and a grayscale option probably covers 95% of all cases?

ndphillips commented:
Thanks for the feedback @hneth ! I agree adding additional arguments to control the individual colors directly could be useful, but I'd prefer to save that for a future PR if needed.

In this PR I have also made a few modifications to DESCRIPTION and NEWS.md to conform to the conventions in the tidyverse (namely https://github.com/tidyverse/dplyr/blob/main/NEWS.md)

* I changed the version to 2.0.0.9000 to avoid the need to increment each development version (i.e.; from 2.0.0.9009 to 2.0.0.90010). I think this will be easier for developers and assuming that most users are installing only stable versions from CRAN, they shouldn't be affected.
* I simplified NEWS.md to not have so many sub-sections and additional descriptors. Again for the purpose of simplicity and to mimic the patterns in the tidyverse.

Happy to discuss reversing these changes if desired
